development/compiling/compiling_for_ios.rst

```diff
@@ -47,9 +47,16 @@ It can be done in three steps, first compile 32 bit version, then compile 64 bit
 
 ::
 
-    $ scons p=iphone tools=no bits=32 target=release arch=arm
-    $ scons p=iphone tools=no bits=64 target=release arch=arm64
-    $ lipo -create bin/godot.iphone.opt.32 bin/godot.iphone.opt.64 -output bin/godot.iphone.opt.universal
+    $ scons p=iphone tools=no target=release arch=arm32
```

Comment:
`arm` was correct, `arm32` is a working alias, but the actual string testing against in `detect.py` (beyond this alias matching) is `arm`. Could be renamed to `armv7` to match what is passed to the compiler, but that's another topic :)

Reply:
I figured the alias could preserve the clarity of 64 bit vs 32 bit while dropping the verbosity of `bits=`. I don't believe Apple has release a v7 device since ~2013 so we maybe we want to consider only supporting 64 bit moving forward?

Reply:
`bits` was unnecessary in the previous example, simply `arch=arm` *is* armv7.

Reply:
And yeah, in the future we could drop `armv7` support, but not for 3.1. You'd be surprised at how much some users want to support old devices (often because that's the ones they have themselves to do tests on).

Reply:
64 bit is iPhone 5s and newer. The iPhone 5c was the last 32 bit handheld device from Apple.

Reply:
Definitely do not want to rock the 3.1 boat 👍

Reply:
I currently support iOS 10 as the minimum. I added more support for the armv7 slices due to this. At least for now, keeping this support really should have no impact on others.

I would imagine once iOS 13 comes out, the desire for iOS 10 support wanes.

Reply:
100% agree, drop 32bit, Apple demands us to compile with the latest XCode anyway if you want to publish on the AppStore. Pretty sure the next major release of XCode won't even support compiling 32bit iOS anymore. It's a dead horse.

Reply:
(3.2+, leave 3.1 as is :) )